packages/patternfly-4/react-core/src/components/Progress/Progress.js

```diff
@@ -71,9 +71,14 @@ class Progress extends Component {
       ...props,
       ...(valueText ? { 'aria-valuetext': valueText } : { 'aria-describedby': `${this.id}-description` })
     };
-    let limitedValue;
-    limitedValue = value < min ? min : value;
-    limitedValue = limitedValue > max ? max : limitedValue;
+    const scaledValue =
+      max === 100
```

Comment:
can you put this in a function and add some test cases?
i tried out with value = 50, min = 50, max = 100 and got result 50 when I expected 0

Reply:
The min param still behaves as a minimum value for progress rather than a scaling factor as how max was changed. Should I also be changing how min works?

Reply:
Changing min parameter based on slack discussion.